src/app/children/child-details/child-details.component.ts

```diff
@@ -15,21 +15,35 @@
  *     along with ndb-core.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import {Component, OnInit} from '@angular/core';
+import {Component, Inject} from '@angular/core';
 import {Child} from '../child';
 import {EntityMapperService} from '../../entity/entity-mapper.service';
 import {Gender} from '../Gender';
-import {ActivatedRoute} from '@angular/router';
+import {ActivatedRoute, Router} from '@angular/router';
+import {FormBuilder, FormGroup, Validators} from "@angular/forms";
+
+declare let require;
 
 @Component({
   selector: 'app-child-details',
   templateUrl: './child-details.component.html',
   styleUrls: ['./child-details.component.css']
 })
-export class ChildDetailsComponent implements OnInit {
-  child = new Child('test');
-  editable = false;   // editable schould be enabled by default and be activated by button
+export class ChildDetailsComponent {
+
+
+  child: Child = new Child("testID");
+
+  creating: boolean = false;  //set true, if new child
+  editable: boolean = false;  //set true, if existing child is edited
+
   gender = Gender;
+
+  genders() : Array<string> {   //Transforms enum into an array for select component
+    let genders = Object.keys(this.gender);
+    return genders.slice(0, genders.length / 2);
+  }
+
   genderSelector;
   selectedGender;
   FamiliyTableSettings = {
```

Comment:
I think family members and socialworkers are currently not supported currently? Then before merging into master to close #44 we should remove these things completely for now.